ci/test_python.sh

```diff
@@ -46,7 +46,7 @@ set +e
 # FIXME: TEMPORARILY disable MG PropertyGraph tests (experimental) tests and
 # bulk sampler IO tests (hangs in CI)
 
-if [[ "${RUNNER_ARCH}" != "ARM64" ]]; then
+if [[ "${RUNNER_ARCH}" != "ARM64" && "${RAPIDS_PY_VERSION}" != "3.13" ]]; then
```

Comment:
Can we skip these with a matrix filter instead? I think currently these spin up a job (for ARM) that immediately ends. Let’s avoid that.